python/langwatch/src/langwatch/langchain.py

```
@@ -0,0 +1,542 @@
+    TraceMetadata,
+    TypedValueChatMessages,
+    TypedValueJson,
+    TypedValueList,
+    TypedValueText,
+)
+
+from uuid import UUID
+from langchain.tools import BaseTool
+
+
+class SpanParams(TypedDict, total=False):
```

Ruff (0.8.2)

Redefinition of unused `SpanParams` from line 25

(F811)